src/qt/bitcoingui.cpp

```diff
@@ -748,6 +748,19 @@ void BitcoinGUI::setNetworkActive(bool networkActive)
     updateNetworkState();
 }
 
+bool BitcoinGUI::updateHeadersSyncProgressLabel()
+{
+    int64_t headersTipTime = clientModel->getHeaderTipTime();
+    int headersTipHeight = clientModel->getHeaderTipHeight();
+    int estHeadersLeft = (GetTime() - headersTipTime)/600;
+    if (estHeadersLeft > 24)
```

Comment:
Comes from https://github.com/bitcoin/bitcoin/pull/8821 over modaloverlay.cpp.
I'll try to move this into a constant.